### PR TITLE
Reviewer Rob - Add alarms for etcd and clearwater-live-verification

### PR DESCRIPTION
**Pull request by bossmc**
Please review these new alarms.

### CODE REVIEW

File: src/alarmdefinition.cpp [hunk 1]
```diff
@@ -543,6 +543,63 @@ namespace AlarmDef {
...
+
+    {CLEARWATER_LIVE_TEST_SERVICE_FAILED, SOFTWARE_ERROR,
+      {
+        {CLEARED,
+          "Service Status: Ok",
+          "The deployment under test has passed live verification, the deployment is healthy."
+        },
+        {INDETERMINATE,
+          "Service Status: Unknown",
+          "The deployment has not completed vefirication yet, the health of the deployment is unknown."
```
Comment (rkd-msw) [inline src/alarmdefinition.cpp]: Typos: "Ok", "vefirication"

File: src/alarmdefinition.cpp [hunk 2]
```diff
@@ -543,6 +543,63 @@ namespace AlarmDef {
...
+            "have occurred."
+        }
+      }
+    },
+
+    {ETCD_CLUSTER_HEALTH, UNDERLYING_RESOURCE_UNAVAILABLE,
+      {
+        {CLEARED,
+          "etcd: Cluster is healthy",
+          "All etcd cluster memenbers are reporting normal operation."
```
Comment (rkd-msw) [inline src/alarmdefinition.cpp]: "memenbers"

File: src/alarmdefinition.cpp [hunk 3]
```diff
@@ -543,6 +543,63 @@ namespace AlarmDef {
...
+        },
+        {MAJOR,
+          "etcd: Cluster node is unhealthy",
+          "etcd is unable to confirm the health of one of its ring nodes. It "
```
Comment (rkd-msw) [inline src/alarmdefinition.cpp]: Is a "ring" the right concept for etcd (and useful to an end user)?

**Comment by rkd-msw** [comment]
Looks fine, apart from typos.

**Comment by bossmc** [comment]
@rkd-msw, can you please add this in to your big cpp-common merge?

**Pull request merged**